pkg/monitoring/rules/rules_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package rules
```

Comment:
please use package rules_test (both files)
all the functions that you are testing are already external so it is easy

Reply:
I had to import rules package for using some functions, but done.